### ISSUE TITLE
Final evaluation feedback

### ISSUE THREAD

**Issue by apapageorgiou** [joomla-projects/gsoc22_accessibility-plugin-script#9]
Review of the requested deliverables to pass the evaluation

### Must have:  
- [x] Fix: Accessibility plugin is not accessible => https://github.com/joomla-projects/gsoc22_accessibility-plugin-script/pull/6
- [x] Fix: Plugin System - Additional Accessibility Features is not correctly detected in the page landmarks by Skip To plugin => https://github.com/joomla-projects/gsoc22_accessibility-plugin-script/pull/10
- [x] Fix: Text to speech - Does not have a stop reading => https://github.com/joomla-projects/gsoc22_accessibility-plugin-script/pull/6
- [x] Fix: Accessibility - Frontend - if "Text to Speech" is activated, the links and some of the buttons don't work => https://github.com/joomla-projects/gsoc22_accessibility-plugin-script/pull/6
- [x] Fix: Accessibility plugin -> Failed to execute 'check' on 'FontFaceSet': Could not resolve '1em Material+Icons' as a font => It's already fixed on https://github.com/joomla/joomla-cms/pull/38009 + Fixes for the emoji icons on https://github.com/joomla-projects/gsoc22_accessibility-plugin-script/pull/12
- [x] Fix: Accessibility button does not appear (at least) on Safari 15 on MacOS 12 Monterey =>It's already fixed on https://github.com/joomla/joomla-cms/pull/38009

### Follow up via the Forum issues
- [x] [Accessibility window missing icons](https://forum.joomla.org/viewtopic.php?f=808&t=992530&sid=0555aea6aaf12b3f463965137eb58f51) =>  It's already fixed on https://github.com/joomla/joomla-cms/pull/38009
- [x] [Accessibility icon has disappeared](https://forum.joomla.org/viewtopic.php?f=811&t=990995) => It's already fixed after updating to 4.1.0 and confirmed by the user
- [x] [Voice and language of Text to speech](https://forum.joomla.org/viewtopic.php?f=808&t=991513) => https://github.com/joomla-projects/gsoc22_accessibility-plugin-script/pull/8 + https://github.com/joomla-projects/gsoc22_accessibility-plugin/pull/5
- [x] [CSS custon for accessibility button](https://forum.joomla.org/viewtopic.php?f=815&t=990619) => User can change the position of a11y after : https://github.com/joomla-projects/gsoc22_accessibility-plugin/pull/3
- [x] [Accessibility Helper](https://forum.joomla.org/viewtopic.php?f=808&t=987981&p=3637798#p3636983) => https://github.com/joomla-projects/gsoc22_accessibility-plugin/pull/3
- [ ] [Questions about Additional Accessibility Features plugin](https://forum.joomla.org/viewtopic.php?f=803&t=981460) => 1. https://github.com/joomla-projects/gsoc22_accessibility-plugin-script/pull/8, 2.
https://github.com/joomla-projects/gsoc22_accessibility-plugin-script/commit/d8d88aa13704cdb23c4cf0a3c7018bf5f5de2d9d

### Optional features
- [x] Line height => https://github.com/joomla-projects/gsoc22_accessibility-plugin-script/pull/5
- [x] Pause animations => https://github.com/joomla-projects/gsoc22_accessibility-plugin-script/pull/4
- [x] Reading velocity => https://github.com/joomla-projects/gsoc22_accessibility-plugin-script/pull/6
- [x] Change the position of the accessibility icon => https://github.com/joomla-projects/gsoc22_accessibility-plugin/pull/3
- [x] Offer the possibility to link to a feedback form and an accessibility statement => https://github.com/joomla-projects/gsoc22_accessibility-plugin-script/pull/7 + https://github.com/joomla-projects/gsoc22_accessibility-plugin/pull/4 + https://github.com/joomla-projects/gsoc22_accessibility-plugin/pull/2

### Bonus
- [x] Dyslexic font => https://github.com/joomla-projects/gsoc22_accessibility-plugin-script/pull/2

**Comment by drmenzelit** [comment]
@Shubhamverma2796 I'm missing a pull request to use the new accessibility icon in Joomla. And the new icon only works if Google Fonts are being used, not with Emojis.

**Comment by Shubhamverma2796** [comment]
which icon are you talking about?

**Comment by drmenzelit** [comment]
![grafik](https://user-images.githubusercontent.com/9153168/188310694-2377e945-faa3-4f24-800f-99281142eda4.png)

**Comment by 1010nishant** [comment]
nice

**Comment by gautam1103** [comment]
I can fix your issue , please assign it to me.

**Comment by shubhangishelar26** [comment]
i want to work on this issues ,please assign me this issue

**Comment by drmenzelit** [comment]
@gautam1103 @shubhshelar there is nothing to do here, that was the evaluation of the GSoC project last year. If you want to contribute to Joomla, please go to github.com/joomla/joomla-cms and search for an issue. You don't need permission to work on something.

**Comment by prokaushik** [comment]
I can fix your issue , please assign it to me.
